### ISSUE TITLE
Heatmap layers should have legend

### ISSUE THREAD

**Issue by doorleyr** [CityScope/CS_cityscopeJS#84]
As discussed in a previous team meeting, the accessibility heatmap layers should have a simple color legend from low to high.
